test/Scanner/rlexe.xml

```diff
@@ -6,6 +6,13 @@
       <compile-flags>-args summary -endargs</compile-flags>
     </default>
   </test>
+  <test>
+    <default>
+      <files>NumericLiteralSuffixRequireICU.js</files>
+      <compile-flags>-args summary -endargs</compile-flags>
+      <tags>exclude_noicu</tags>
```

Comment:
maybe prefer to name this RequireIntl? We do still support winglob, and the test works with winglob. I think this could also be achieved with `if (WScript.Platform.INTL_LIBRARY === "winglob" || WScript.Platform.INTL_LIBRARY === "icu")`

Reply:
This is what I was looking for, thanks for the heads-up.